guides/common/modules/proc_using-container-registries.adoc

```diff
@@ -2,17 +2,33 @@
 = Using container registries
 
 Podman and Docker can be used to fetch content from container registries.
+Containers are pushed to {Project}'s container registry.
+The {Project} registry follows the Open Containers Initiative (OCI) specification, so you can push content to Katello by using the same methods that apply to other registries.
+For more information about OCI, see https://opencontainers.org/[Open Container Initiative Distribution Specification].
+
+.Prerequisites
+* To push content to an organization or product, a user must have permissions to create repositories.
 
 ifndef::orcharhino[]
 .Container registries on {SmartProxies}
-
 On {SmartProxies} with content, the https://github.com/Katello/smart_proxy_container_gateway[Container Gateway] {SmartProxy} plugin acts as the container registry.
 It caches authentication information from Katello and proxies incoming requests to Pulp.
 The Container Gateway is available by default on {SmartProxies} with content.
 endif::[]
 
-.Procedure
+.Limitations on container registries
+* A repository in Katello is created after the content is pushed.
+* Content is pulled as soon as it is pushed.
```

Comment:
This was supposed to be:
```suggestion
* Content is pullable as soon as it is pushed.
```
which was just a way of saying that there shouldn't be any delay between pushing content and being able to pull it. I think it might be extra info that's not needed in the docs.